docs/digging-deeper/advanced-markdown.md

```diff
@@ -160,6 +160,47 @@ anything within the path label will be rendered as HTML. This means you can add
 
 The filepaths are hidden on mobile devices using CSS to prevent them from overlapping with the code block.
 
+
+## Dynamic Markdown Links
+
+HydePHP provides a powerful feature for automatically resolving dynamic links within your pages and posts using a special Hyde Markdown link syntax, designed to match the syntax of the `Hyde` facade.
+
+### Usage
+
+You can use the following syntax options in your Markdown files:
+
+```markdown
+<!-- Resolving a page route -->
+[Home](hyde::route('home'))
+
+<!-- Resolving a relative link -->
+[About](hyde::relativeLink('about'))
```

Comment:
Can't see too big of a use case for this, doesn't really offer any value.

Reply:
Removed in https://github.com/hydephp/develop/commit/9589bc17d1165cbc577e298d4500c7aeb9a2c48d